src/posts/posts.routes.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const PostRoute = express.Router();
+const postController = require("./posts.controller");
+const postService = require("./posts.service");
```

Comment:
Your service should not talk to your route

route=>controller=>service

Reply:
Insightful. The change has been effected.